services/kube-prometheus-stack/63.0.0/defaults/cm.yaml

```diff
@@ -355,9 +355,9 @@ data:
         pathType: ImplementationSpecific
       sidecar:
         image:
-          registry: ghcr.io
-          repository: mesosphere/dkp-container-images/quay.io/kiwigrid/k8s-sidecar
-          tag: 1.26.1-d2iq.1
+          registry: docker.io
+          repository: docker.io/kiwigrid/k8s-sidecar
```

Comment:
The reason why airgapped tests are failing is that `docker.io` is here twice, in `registry:` and also part of `repository:`.